src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBUtils.java

```diff
@@ -36,11 +37,12 @@
      * @param callsetJson path to the GenomicsDB callset JSON
      * @param vidmapJson path to the GenomicsDB vidmap JSON
      * @param vcfHeader VCF with header information to use for header lines on export
+     * @param doGnarlyGenotyping set genotyping parameters as for the GnarlyGenotyper tool
```

Comment:
Can you pass GenomicsDBOptions here instead of the evaluated boolean `doGnarlyGenotyping`? GenomicsDBOptions will probably have to be extended to override GenomicsDBExportConfiguration.ExportConfiguration.newBuilder settings that are hardcoded now, e.g.
ProduceGTField(defaults to false)
ProduceGTWithMinPLValueForSpanningDeletions(defaults to false), 
SitesOnlyQuery(defaults to false)
MaxDiploidAltAllelesThatCanBeGenotyped(defaults to GenotypeLikelihoods.MAX_DIPLOID_ALT_ALLELES_THAT_CAN_BE_GENOTYPED), etc.

Reply:
Excellent point.  That helps clean things up a lot.